src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -63,7 +63,7 @@ public async Task<ScanResult> LoadAsync(string[] args, CancellationToken cancell
 
         var logFile = Path.Combine(
             baseArguments.Output ?? Path.GetTempPath(),
-            $"GovCompDisc_Log{DateTime.Now:yyyyMMddHHmmssfff}.log");
+            $"GovCompDisc_Log_{Environment.ProcessId}_{DateTime.Now:yyyyMMddHHmmssfff}.log");
```

Comment:
Don't we need the timestamp first? Otherwise it's `GovCompDisc_Log_12345_20230306...` instead of `GovCompDisc_Log_20230306_12345...`

Reply:
Ah yes you're right.